PropertyChanged.Fody.Analyzer/PropertyChanged.Fody.Analyzer.csproj

```diff
@@ -7,7 +7,7 @@
   </PropertyGroup>
   <ItemGroup>
     <PackageReference Include="Microsoft.CodeAnalysis.Analyzers" Version="3.3.4" PrivateAssets="all" />
-    <PackageReference Include="Microsoft.CodeAnalysis.CSharp" Version="4.3.0" PrivateAssets="All" />
-    <PackageReference Include="Nullable.Extended.Analyzer" Version="1.15.6169" PrivateAssets="all" />
+    <PackageReference Include="Microsoft.CodeAnalysis.CSharp" Version="4.8.0" PrivateAssets="All" />
```

Comment:
Oh, I didn't notice this earlier, but I think this dependency should be rolled back to v4.0.1 and frozen at that version unless there's a good reason to use a newer one, since referencing a newer version requires the user to have a .NET SDK or Visual Studio version which supports it. See [here](https://github.com/dotnet/roslyn/blob/main/docs/wiki/NuGet-packages.md) for the version map.

The problem is that it won't play along with tools which always suggest to bump the packages to the latest versions. 😞